src/main/cpp/apple_fsnotifier.cpp

```diff
@@ -25,7 +25,13 @@ typedef struct watch_details {
     CFRunLoopRef threadLoop;
 } watch_details_t;
 
-static void reportEvent(const char *event, char *path, jobject watcherCallback) {
+static jobject getTypeEnum(JNIEnv *env, const char *name) {
```

Comment:
generally, what we do elsewhere is do this lookup in Java land and instead just pass the ordinals for the constants across the JNI boundary.

Reply:
Changed it so in #53. Thanks for the review.